content/quickstart.md

```diff
@@ -60,6 +60,14 @@ Depending on your role please use the appropriate template in the instructions b
 
 ### Deployment instructions
 
+Before we can start installing Syndesis we have to add some policies
+
+```bash
+$ oc login -u system:admin
+$ oc adm policy add-cluster-role-to-user system:auth-delegator -z syndesis-oauth-client
+$ oc login -u developer
+```
+
```

Comment:
@KurtStam @zregvart I thought we reverted this requirement ?

Reply:
Yeah we did by removing the functionality 😄

Reply:
I think the 2 operations (requirement removal and doc update) happened more or less at the same time.

Reply:
@KurtStam so can we please remove this section again ?